templates/patterns/header/pattern-header--variant-eu.html.twig

```diff
@@ -7,7 +7,11 @@
       </ul>
     </div>
   </nav>
-  <div class="bcl-header__project">
+  {% set _project_classes = 'bcl-header__project' %}
```

Comment:
I think we have a lot of duplicate code here, lets take the chance and use extend template and block content